bf_cpp_downloads.html

```diff
@@ -202,38 +164,146 @@
+                        <tbody>
+                            <tr>
+                                <th width="269">Downloads</th>
+                                <th width="85">Size</th>
+                                <th width="150">File Name</th>
+                                <th width="125">SHA1 Checksum</th>
+                            </tr>
+                            <tr>
+                                <td><a href="@UBUNTU1404PLATFORM_RELEASE@"><img
+                                        src="images/bf_c++_ubuntu_14_04_release.png"
+                                        alt="Ubuntu 14.04 C++ Release binaries"
+                                     /></a></td>
+                                <td align="center">@UBUNTU1404PLATFORM_RELEASE_SIZE@</td>
+                                <td>@UBUNTU1404PLATFORM_RELEASE_BASE@</td>
+                                <td align="center">@UBUNTU1404PLATFORM_RELEASE_SHA1@ (<a
+                                        href="@UBUNTU1404PLATFORM_RELEASE@.sha1">SHA1</a>)</td>
+                             </tr>
+                             <tr>
+                                <td><a href="@UBUNTU1404PLATFORM_DEBUG@"><img
+                                        src="images/bf_c++_ubuntu_14_04_debug.png"
+                                        alt="Ubuntu 14.04 C++ Debug binaries"
+                                     /></a></td>
+                                <td align="center">@UBUNTU1404PLATFORM_DEBUG_SIZE@</td>
+                                <td>@UBUNTU1404PLATFORM_DEBUG_BASE@</td>
+                                <td align="center">@UBUNTU1404PLATFORM_DEBUG_SHA1@ (<a
+                                        href="@UBUNTU1404PLATFORM_DEBUG@.sha1">SHA1</a>)</td>
+                             </tr>
+                        </tbody>
+                    </table>
+                </div>
+                <ul>
                     <li>
-                        <p>In all cases you should be able to build
-			   from source to work on operating systems
-			   which meet
-			   the <a href="@DOC_URL@/developers/cpp/overview.html">requirements</a>.
-			   This will ensure your build is compatible
-			   with the libraries available on the
-			   system.</p>
+                        <p>These releases are built against a specific
+                           platform release and therefore should
+                           always work with the shared libraries
+                           provided by the platform.  These releases
+                           are provided only for platforms with
+                           guaranteed ABI stability during their
+                           supported lifetime, and which also provide
+                           all of the needed build dependencies.</p>
                     </li>
                 </ul>
                 <h2><a name="api" id="api"></a>Bio-Formats C++ API
```

Comment:
Image name for API docs icon below needs changing to use new icon:
bf_api_docs.png -> bf_api_doc.png